### PR TITLE
Update teams-custom-app-policies-and-settings.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2317
Added link for App Studio

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-custom-app-policies-and-settings.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ ROBOTS: NOINDEX, NOFOLLOW
 > [!NOTE]
 > For the current method of managing apps in Microsoft Teams, see [Manage Microsoft Teams settings for your organization](https://docs.microsoft.com/MicrosoftTeams/enable-features-office-365).
 
+> [!NOTE]
```
Comment (nenonix) [inline Teams/teams-custom-app-policies-and-settings.md]:
```suggestion
> [!TIP]
```

File: Teams/teams-custom-app-policies-and-settings.md [hunk 2]
```diff
@@ -27,6 +27,10 @@ ROBOTS: NOINDEX, NOFOLLOW
...
+> To use App Studio see [Get started on the Microsoft Teams platform with C#/.NET and App Studio](https://docs.microsoft.com/microsoftteams/platform/get-started/get-started-dotnet-app-studio)
+The last step is not working yet, so you will need to download the zip and install it the old way at [Upload an app package to Microsoft Teams](https://docs.microsoft.com/microsoftteams/platform/concepts/apps/apps-upload).
```
Comment (nenonix) [inline Teams/teams-custom-app-policies-and-settings.md]: Suggest that this is left as a note on the page and following the step that is not working yet as "If this step doesn't work for you, you can download the zip file and install it manually. See [Upload an app package to Microsoft Teams](https://docs.microsoft.com/microsoftteams/platform/concepts/apps/apps-upload) for more information." or something like that.

File: Teams/teams-custom-app-policies-and-settings.md [hunk 3]
```diff
@@ -27,6 +27,10 @@ ROBOTS: NOINDEX, NOFOLLOW
 > [!NOTE]
 > For the current method of managing apps in Microsoft Teams, see [Manage Microsoft Teams settings for your organization](https://docs.microsoft.com/MicrosoftTeams/enable-features-office-365).
 
+> [!NOTE]
+> To use App Studio see [Get started on the Microsoft Teams platform with C#/.NET and App Studio](https://docs.microsoft.com/microsoftteams/platform/get-started/get-started-dotnet-app-studio)
```
Comment (nenonix) [inline Teams/teams-custom-app-policies-and-settings.md]:
```suggestion
> To use App Studio see [Get started on the Microsoft Teams platform with C#/.NET and App Studio](https://docs.microsoft.com/microsoftteams/platform/get-started/get-started-dotnet-app-studio).
```

Comment (JohanFreelancer9) [inline Teams/teams-custom-app-policies-and-settings.md]:
```suggestion
> To use App Studio, see [Get started on the Microsoft Teams platform with C#/.NET and App Studio](https://docs.microsoft.com/microsoftteams/platform/get-started/get-started-dotnet-app-studio).
```

**Pull request merged**